src/index.html

```diff
@@ -6,18 +6,22 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
 	<title>Update your title here</title>
-	<!--
-		Webpack will automatically link your CSS file for you.
-		Work in the src/css/styles.scss file.
-	-->
 </head>
 <body>
-	<main class="content-container">
-		<h1 class="page-heading">Welcome to your final project!</h1>
-	</main>
-	<!--
-		Webpack will automatically link your JS file for you.
-		Work in the src/js folder.
-	-->
+	<div class="header-container">
+      <h1 class="logo">Mix&Match</h1>
+      <button class="search">search</button>
+      <button class="hamburger">hamburger</button>
+   </div>
```

Comment:
I think you and Hang both did styles for this area. Be sure to coordinate before merging or you might be in for an unpleasant surprise - both your pages should have the same markup and styles :)

Reply:
Also, you may want to make this a `<nav>` element :)